circle.yml

```diff
@@ -13,4 +13,4 @@ deployment:
       - git config --global user.name "Apiary Bot"
       - git config --global user.email "support@apiary.io"
       - git remote set-url origin "https://${GH_TOKEN}@github.com/apiaryio/apiaryio.github.com"
-      - rake deploy
+      - bundle exec rake deploy
```

Comment:
I think that error: ```rake deploy
rake aborted!
LoadError: cannot load such file -- html-proofer
/home/ubuntu/apiaryio.github.com/Rakefile:2:in `<top (required)>'
(See full trace by running task with --trace)

rake deploy returned exit code 1

Action failed: rake deploy``` is that can't find gem, but gem is installed in step before. This can help, maybe more versions ruby on machine and this looks as some path problem.